src/store/mutations.js

```diff
@@ -4,6 +4,29 @@ import { findDirectory } from '../plugins/helpers';
 import { colorArray, colorChamber, colorHeaterBed/*, temperaturChartSampleLength*/ } from './variables';
 import defaultPrinter from './printer';
 
+let consoleFilterCache = [];
+
+function updateConsoleFilterCache(filters) {
```

Comment:
you can move this function to plugins/helpers and only import these into the mutations